hraven-core/src/test/java/com/twitter/hraven/datasource/TestJobHistoryService.java

```diff
@@ -342,19 +345,17 @@ public void testRemoveJob() throws Exception {
   private void assertFoundOnce(byte[] column, Put jobPut, int expectedSize,
 		  String expectedValue) {
 	  boolean foundUserName = false;
-/*TODO dogpiledays
-    List<KeyValue> kv1 = jobPut.get(Constants.INFO_FAM_BYTES, column);
-	  assertEquals(expectedSize, kv1.size());
-	  for (KeyValue kv : kv1) {
-		assertEquals(Bytes.toString(kv.getValue()), expectedValue);
-	    // ensure we don't see the same put twice
-		assertFalse(foundUserName);
-		// now set this to true
-		foundUserName = true;
-  	  }
-  */    // ensure that we got the user name
-    // TODO dogpiledays
-    foundUserName = true;
+
+    List<Cell> cells = jobPut.get(Constants.INFO_FAM_BYTES, column);
+	  assertEquals(expectedSize, cells.size());
+	  for (Cell cell : cells) {
+      assertEquals(expectedValue, Bytes.toString(CellUtil.cloneValue(cell)));
```

Comment:
I think the indentation may be a bit off, but it's nothing major. I will edit it later when I am updating things for a final merge.